localization/english/wc_game_concepts_l_english.yml

```diff
@@ -0,0 +1,10 @@
+﻿ l_english:
+     #WARCRAFT#
+     game_concept_caster:0 "Caster"
+     game_concept_caster_desc:0 "Casters are magic weilding [men_at_arms|E] which are lightly armored but can deal heavy damage.\n\nWhile each kind of Caster is unique, they generally [counter|E] [massive_infantry|E],[heavy_infantry|E] and [pikemen|E], and are in turn Countered by [heavy_infantry|E],[light_cavalry|E] and [heavy_cavalry|E] .\n\nCasters are also often strong on flat or hilly [terrain|E].
```

Comment:
The description doesn't work.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/105865930-d8e9b100-600c-11eb-9ec5-7fdea88d8c86.png)

</details>